README.md

```diff
@@ -1,2 +1,12 @@
-# hw01
-Repository for homework 1
+#Carman Fowler Biography
```

Comment:
A space between the pound sign and the text should make this appear properly in GitHub! I see that it appeared correctly in the preview in RStudio, but not here.

Reply:
That is something I struggled with as well. I thought I was going crazy until I sent in a commit and realized the preview was a little bit off.